django/contrib/admin/static/admin/css/base.css

```diff
@@ -37,7 +90,7 @@ a img {
 }
 
 a.section:link, a.section:visited {
-    color: #fff;
+    color: var(--body-bg);
```

Comment:
Hello,
Sorry. for the late feedback. I started working on updating https://github.com/django/django/pull/12444 and immediately encountered and issue. I want to make the caption dark with a light text as in https://github.com/django/django/pull/12444#issuecomment-585297798, but it's not possible with the current variables. Could you please split the colors a bit more? If not, I could make a separate PR for that. Anyway, using `body-bg` for things that are not a background seems strange.

Reply:
@mimi89999 No stress. Add an initial commit to your PR adding the variable. (Finding this sort of this is expected, and whole point of the pre-release phase. 😄) Thanks!